classes/Provider/ApplicationServiceProvider.php

```diff
@@ -91,4 +94,11 @@ public function register(Container $app)
             return new PseudoRandomStringGenerator();
         };
     }
+
+    public function subscribe(Container $app, EventDispatcherInterface $dispatcher)
+    {
+        $dispatcher->addSubscriber(new DatabaseSetupListener(
+            $app[Capsule::class]
```

Comment:
Since we only pass in  a single argument to the constructor of  `DatabaseSetupListener`, can we just instantiate the latter in a single line?

```php
$dispatcher->addSubscriber(new DatabaseSetupListener($app[Capsule::class]));
```

Reply:
Sure.